events/tests/conftest.py

```diff
@@ -504,22 +514,51 @@ def keyword_id(data_source, organization, kw_name, make_keyword_id):
     return make_keyword_id(data_source, organization, kw_name)
 
 
+@pytest.mark.django_db
+@pytest.fixture(scope="class")
+def make_keyword_set():
```

Comment:
This can be a fixture, but afaik this could also just be test utility function e.g. in `events.tests.utils`. :shrug: 

```python
def make_keyword_set(data_source, organization, kw_set_name):
    return KeywordSet.objects.create(
        id=data_source.id + ":" + kw_set_name,
        name=kw_set_name,
        organization=organization,
        data_source=data_source,
    )
```